src/rdkafka.c

```diff
@@ -1117,6 +1172,10 @@ static void rd_kafka_stats_emit_all (rd_kafka_t *rk) {
 
 		rd_kafka_broker_unlock(rkb);
 
+                rd_avg_destroy(&int_latency);
```

Comment:
were these errantly omitted previously?

Reply:
They weren't needed before because rd_avg_t had no allocations prior to the inclusion of the histogram.